src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -676,27 +985,32 @@ def svm_help(schema_madlib, message, is_svc, **kwargs):
 def svm(schema_madlib, source_table, model_table,
         dependent_varname, independent_varname, kernel_func,
         kernel_params, grouping_col, params, is_svc,
-        verbose, detect_novelty=False, **kwargs):
+        verbose, add_intercept=False, **kwargs):
```

Comment:
I agree that it is probably ok for now that we discourage the use of `array[1, ...]` only in the kernel case and ask users to specify `fit_intercept` in the kernel params. We will also need to add notes in the doc to make it clear to the users.